recipes/m17-cxx-demod/bld.bat

```diff
@@ -0,0 +1,28 @@
+setlocal EnableDelayedExpansion
+@echo on
+
+:: Make a build folder and change to it
+mkdir build
+cd build
+
+:: configure
+cmake -G "Ninja" ^
+    -DCMAKE_BUILD_TYPE:STRING=Release ^
+    -DCMAKE_INSTALL_PREFIX:PATH="%LIBRARY_PREFIX%" ^
+    -DCMAKE_PREFIX_PATH:PATH="%LIBRARY_PREFIX%" ^
+    -DENABLE_DOXYGEN=OFF ^
+    -DENABLE_TESTING=ON ^
```

Comment:
```suggestion

```
This package does nothing with these flags (particular to GNU Radio usually), so you can remove them.

Reply:
Accepted.